### ISSUE TITLE
New data project.

### ISSUE THREAD

**Issue by lvisintini** [kingargyle/iaskirmish-data#2]
I made the first release for [imperial-assault-data](https://github.com/lvisintini/imperial-assault-data) yesterday and someone directed me here.

It seems I have been working on a project like yours for a few months now.

[imperial-assault-data](https://github.com/lvisintini/imperial-assault-data) has data and images for all the components for the game.

I'm planning on adding JSON Schema tests and to add more data to the files in the near future

Would you be able to review the contents imperial-assault-data?
Perhaps the new data would allow you to tackle other features for xstreamer or other projects

**Comment by kingargyle** [comment]
Yeah, sorry for the late response.   I'll take a look.  If I can leverage your project I'm more than willing to do so, as I won't have to maintain the data files.
